package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "collections",
+  "description": "Serves GOV.UK navigation pages, browse, topic, step-by-steps & services and information pages.",
+  "private": true,
```

Comment:
I guess there's no harm in this attribute being here but is this appropriate as this repo itself is public? Sorry if I've misunderstood the intent here.

Reply:
By setting it to private we express the fact that this project is not meant to be published via npm (npm checks for this value before allowing code to be published).